static/security.txt

```diff
@@ -0,0 +1,20 @@
+# This file provides security vulnerability reporting information for the AlmaLinux OS project.
+# Please follow the criteria below to ensure your report reaches the correct team.
+Preferred-Languages: en
+
+# Use this contact if the issue is directly related to the AlmaLinux operating system itself.
+Contact: https://bugs.almalinux.org
+
+# Use this contact for vulnerabilities that are associated with the AlmaLinux project or its infrastructure.
+Contact: mailto:security@almalinux.org
```

Comment:
```suggestion
# Use this contact for vulnerabilities that require coordinated disclosure, no matter in which part of the project they are found. 
Contact: security@almalinux.org

# Use this contact if the issue is directly related to the AlmaLinux operating system itself but does not require coordinated disclosure.
Contact: https://bugs.almalinux.org

# For any other flaws, please report them in the repository that is associated with the part of the project in question.

```
I dunno if we can leave that last one open ended, according to the RFC. If we can't, let's just have folks email security.

Reply:
This would be considered a comment so I think we're good.